packaging/docker/README.md

```diff
@@ -56,10 +53,15 @@ If you want to have your container names resolved by netdata it needs to have ac
 grep docker /etc/group | cut -d ':' -f 3
 ```
 
+### Pass command line options to Netdata 
+
+Since we use an [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) directive, you can provide [netdata daemon command line options](https://docs.netdata.cloud/daemon/#command-line-options) such as the IP address netdata will be running on, using the [CMD instruction](https://docs.docker.com/engine/reference/builder/#cmd). 
```

Comment:
"using the [command instrcution]" would be better. `CMD` suggests to create new dockerfile, `command` is more generic.